demo/src/main/java/com/novoda/merlin/demo/presentation/DemoActivity.java

```diff
@@ -45,6 +48,18 @@ private void showToast(int toastText) {
         Toast.makeText(this, toastText, Toast.LENGTH_SHORT).show();
     }
 
+    private final View.OnClickListener wifiConnectedOnClick = new View.OnClickListener() {
+
+        @Override
+        public void onClick(View view) {
+            if (merlinsBeard.isConnectedTo(WIFI)) {
+                showToast(R.string.toast_connected);
+            } else {
+                showToast(R.string.toast_disconnected);
```

Comment:
The action button says `is wifi connected` and the reply would be `disconnected`